### PR TITLE
feat(selectv2): added css variables

### PR DESCRIPTION
**Pull request by bobbonius**
# Description

Added css variables to a variety of elements in order to more easily control styling in custom components

### CODE REVIEW

File: src/components/SelectV2/Select.tsx [hunk 1]
```diff
@@ -78,18 +78,22 @@ interface DisplayValueProps {
 
 const DisplayValue = styled.span<DisplayValueProps>`
   grid-area: 1 / 1 / 1 / 4;
-  font-size: ${fontSize[18]};
-  line-height: ${space[56]};
-  block-size: ${space[56]};
-  padding-inline: ${space[16]} ${space[48]};
-  color: ${color.foreground};
+  font-size: var(--displayValueFontSize, ${fontSize[18]});
```
Comment (RobVermeer) [inline src/components/SelectV2/Select.tsx]: I would prefix this with Select, since `displayValue` can be anywhere

Comment (bobbonius) [reply src/components/SelectV2/Select.tsx]: Good idea

**Review by RobVermeer — APPROVED** [review]
✅

**Pull request merged**